querybook/server/tasks/run_query.py

```diff
@@ -56,6 +56,7 @@ def run_query_task(self, query_execution_id):
                 query_execution_id, executor, error_message, session=session
             )
             notifiy_on_execution_completion(query_execution_id, session=session)
+            qe_logic.update_es_query_execution_by_id(query_execution_id)
```

Comment:
given you are already in a celery task, you can call the logic directly to update instead of spawning a new one